plugins/LilyPad/Tooltips.cpp

```diff
@@ -63,8 +63,11 @@ LPWSTR dialog_message(int ID, bool *updateText)
                    L"Double-click a device in the list or right-click it and select \"Test Device\" to display a continuously updated list of the state of all inputs on the selected device.\n"
                    L"Use this option to check if all the inputs on a controller function properly.\n\n"
                    L"Right-click and select \"Refresh\" to update the list of devices in case a recently connected device has not shown up yet.";
-        case IDC_G_XI:
-            return L"Xbox 360/Xbox One controllers (or devices/software imitating such) only";
+        case IDC_G_DI:
+	        return L"(Legacy) Enable this if your gamepad doesn't support Xinput.";
+	    case IDC_G_XI:
+            return L"For Xbox 360/ Xbox One controllers (or devices supporting Xinput).\n\n"
```

Comment:
Should use spaces, also spacing is wrong. Above line as well.

Reply:
I have used spaces though in the correct format:
![image](https://user-images.githubusercontent.com/24227051/91743777-a2704d00-ebb8-11ea-9952-1128ec6769bf.png)

Reply:
ctrl + r + w to show spacing marks. Tabs will be marked with arrows, spaces with dots.